docs/docs/experts/index.html

```diff
@@ -0,0 +1,6 @@
+<!doctype html><html lang=en><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><title>Experts
```

Comment:
(I'm only commenting on this file and not the original because GitHub won't let me)

We should find a better way to present the experts map than linking to a `toml` file that users' browsers will download.